Empire/PrefillAdsInjector.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Empire;
+
+
+class PrefillAdsInjector {
+    private $adsConfig;
+    private $prefillConfig;
+    private $targeting;
+
+    public function __construct( $adsConfig, $prefillConfig, $targeting ) {
```

Comment:
If we aren't making a class out of the AdsConfig, then we should at least document the expected keys / keys we depend on and what should be in them.